src/main.ts

```diff
@@ -39,7 +39,7 @@ export async function transformMain(
 
   // feature information
   const hasScoped = descriptor.styles.some(s => s.scoped)
-  const hasCssModules = descriptor.styles.some(s => s.module)
+  const hasCssModules = descriptor.styles.some(s => s.module && !!s.content && !!s.content.trim())
```

Comment:
Creating a trimmed copy of the string to see if it is empty isn't a very fast way to check to see if the string is only whitespace. The current approach you are using appears to be about 40% slower than:
```ts
const hasNonWhitespaceContent = (content: string | undefined) => !!content && /\S/.test(content)
```